tomviz/ModuleMeasurementCubeWidget.h

```diff
@@ -0,0 +1,79 @@
+/******************************************************************************
+
+  This source file is part of the tomviz project.
+
+  Copyright Kitware, Inc.
+
+  This source code is released under the New BSD License, (the "License").
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+******************************************************************************/
+#ifndef tomvizModuleMeasurementCubeWidget_h
+#define tomvizModuleMeasurementCubeWidget_h
+
+#include <memory>
+
+#include <QSharedPointer>
+#include <QWidget>
+
+/**
+ * \brief UI layer of ModuleMeasurementCube.
+ *
+ * Signals are forwarded to the actual actuators in ModuleMeasurementCube.
+ * This class is intended to contain only logic related to UI actions.
+ */
+
+namespace Ui {
+class ModuleMeasurementCubeWidget;
+}
+
+namespace tomviz {
+
+class ModuleMeasurementCubeWidget : public QWidget
+{
+  Q_OBJECT
+
+public:
+  ModuleMeasurementCubeWidget(QWidget* parent_ = nullptr);
+  ~ModuleMeasurementCubeWidget() = default;
```

Comment:
Yeah - you need this to be in the implementation in order to prevent it being inlined for the QScopedPointer